### PR TITLE
MSI: Always with logfile #6609

### PR DESCRIPTION
**Pull request by guruz**
* [ ] needs cherry-pick into 2.5 after

### CODE REVIEW

File: src/gui/updater/ocupdater.cpp [hunk 1]
```diff
@@ -283,9 +283,11 @@ void OCUpdater::slotStartInstaller()
             return QDir::toNativeSeparators(path);
         };
 
-        QString command = QString("&{msiexec /norestart /passive /i '%1'| Out-Null ; &'%2'}")
-            .arg(preparePathForPowershell(updateFile))
-            .arg(preparePathForPowershell(QCoreApplication::applicationFilePath()));
+        auto msiLogFile = cfg.configPath() + "msi.log";
```
Comment (ckamm) [inline src/gui/updater/ocupdater.cpp]: Is `configPath()` here a good choice for windows?

Comment (dschmidt) [reply src/gui/updater/ocupdater.cpp]: I'd say so: it's next to the .msi file itself

**Comment by guruz** [comment]
I can't push the patch into the PR https://github.com/owncloud/client/pull/6578 because it's in @dschmidt 's repo instead of here in owncloud/client :-/ @dschmidt can you do it?

**Comment by dschmidt** [comment]
Yes, sure. Will do it later today

**Pull request merged**